roles/piper/templates/gen_GATK_ref.sh

```diff
@@ -0,0 +1,13 @@
+module load bioinfo-tools
```

Comment:
A "good habit" is to name all templates with the `.j2` file extension (short for jinja2, the expression engine used for e.g. templates), and then when you're deploying them in your task you rename the file in the `dest` argument to be without the `j2` part.